lib/ansible/modules/cloud/ovirt/ovirt_template.py

```diff
@@ -426,6 +461,23 @@ def _get_vnic_profile_mappings(module):
     return vnicProfileMappings
 
 
+def find_subversion_template(module, templates_service):
+    version = module.params.get('version')
+    templates = templates_service.list()
+    for template in templates:
+        if version.get('number') == template.version.version_number and module.params.get('name') == template.name:
+            return template
+
+    # when user puts version number which does not exist
+    raise ValueError(
```

Comment:
Well actually, should we in this case return new version of that template, rather then raise Exception?

Reply:
Please ignore, this is called when non-existin number is passed. So it's ok